packages/manager/modules/pci/src/projects/project/data-processing/notebooks/add-notebook/command/command.controller.js

```diff
@@ -0,0 +1,16 @@
+export default class CommandCtrl {
+  /* @ngInject */
+  constructor($translate) {
+    this.$translate = $translate;
```

Comment:
I don't see any reference to $translate, do you need to import it ?

Reply:
Removed :)